python/orca/src/bigdl/orca/automl/model/base_keras_model.py

```diff
@@ -202,6 +208,9 @@ def state_dict(self):
     def load_state_dict(self, state):
         self.config = state["config"]
         self.model = self.model_creator(self.config)
+        if self.model.layers[0].weights.__len__() == 0:
+            input_shape = self._get_input_shape()
+            _, = self.model(input_shape)
```

Comment:
leave this bug to following PR. We may raise an issue first.

Reply:
Ok, I temporarily removed the changes